app/src/main/java/org/simple/clinic/phone/MaskedPhoneCallerImplementation.kt

```diff
@@ -1,12 +1,11 @@
 package org.simple.clinic.phone
 
-import android.net.Uri
 import io.reactivex.Completable
 import io.reactivex.Single
 import org.simple.clinic.activity.TheActivity
 import javax.inject.Inject
 
-class TwilioMaskedPhoneCaller @Inject constructor(
+class MaskedPhoneCallerImplementation @Inject constructor(
```

Comment:
Just a suggestions: should we name this `OfflineMaskedPhoneCaller` instead?

Reply:
Done